dll/win32/comctl32/listview.c

```diff
@@ -9584,7 +9584,7 @@ static BOOL LISTVIEW_DrawTrackLine(const LISTVIEW_INFO *infoPtr)
 /***
  * DESCRIPTION:
  * Called when an edit control should be displayed. This function is called after
- * we are sure that there was a single click - not a double click (this is a TIMERPROC).
+ * we are sure that there was a single click - not a double-click (this is a TIMERPROC).
  *
  * PARAMETER(S):
  * [I] hwnd : Handle to the listview
```

Comment:
This is 3rd-party code, please send the fix upstream.

https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/WINESYNC.txt;hb=50774f865c4f9d9b926f32a8bcfc20bbeced8fb5#l59